eng/targets/ReferenceAssembly.targets

```diff
@@ -51,6 +51,7 @@
     <PropertyGroup>
       <_RefSourceFileTFM>$(TargetFramework)</_RefSourceFileTFM>
       <_RefSourceFileTFM Condition="$(TargetFramework.StartsWith('netcoreapp'))">netcoreapp</_RefSourceFileTFM>
+      <_RefSourceFileTFM Condition=" '$(TargetFramework)' == 'net5.0' ">netcoreapp</_RefSourceFileTFM>
```

Comment:
I suspect there's a future-proof way to do this. Any suggestions @ViktorHofer?

Reply:
Yes, you should read from TargetFrameworkIdentifier and TargetFrameworkVersion which are available in targets files. TargetFramework should only be read in the project file for e.g. conditions as it's a "user-facing" property which can and will be overloaded going forward.

cc @dsplaisted

Reply:
Here's what you probably want if you want a condition that will tell you if you are on .NET 5 or greater:

```
Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' And
           $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '5.0'))"
```